TrelloAutotest/Tests/BoardSearchTest.cs

```diff
@@ -8,6 +9,15 @@ class BoardSearchTest : BaseTest
     {
         private const string _newBoardName = "Nowa tablica";
 
+        [OneTimeSetUp]
+        public void OneTimeSetUpSetUp()
+        {
+            BaseRestClient.ClientConnection("https://api.trello.com/1/");
+
+            var POSTRequests = new POSTRequests();
+            POSTRequests.CreateBoard(_newBoardName);
```

Comment:
Can this body be in a separate class like Api, and called like Api.CreateBoard(boardname)? In the test class, we don't want to see the implementation. Just chains.